docs/apm/traces/working-with-tracing-data/services-list-map.md

```diff
@@ -0,0 +1,117 @@
+---
+id: services-list-map
+title: Services List and Map
+description: View your application topology and investigate microservice interactions.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+To open the Services List and Map, click **+ New** > **Services**.
+
+<img src={useBaseUrl('img/traces/open-service-map.png')} alt="open servicemap" width="400"/>
+
+## Services List view
+
+The **Services List** view provides a compact, high-level overview of your application service health insights and KPIs. You can filter and sort your KPI data using the filters and column headers.
+
+![services-list-display](/img/traces/services-list-filters.png)
+
+KPIs listed in the table reflect the previous 15 minutes of data. Clicking on any service will open the Entities panel allowing for further drill-down.
+
+### General Settings
+
+Configure anomaly detection in **General Settings** ([learn more](#anomaly-detection)).
+
+<img src={useBaseUrl('img/traces/services-list-settings.png')} alt="services-list-settings" />
+
+
+:::note
+Your Services List and Map settings are specific to you (not your team) and will be preserved in your browser's local storage.
+:::
+
+### Display Settings
+
+Display and hide columns using **Display Settings**.
+
+![services-list-display](/img/traces/services-list-display.png)
+
+
+## Services Map view
+
+The **Services Map** view visually maps out your application environment, giving you a greater understanding of your application architecture, hierarchy, and dependencies between monitored microservices. Health and load of each microservice is reflected in size and color to help you spot potential problems and bottlenecks in your application infrastructure.
+
+Maps are built automatically from distributed tracing data in real time as it arrives in Sumo Logic.
+
+<img src={useBaseUrl('img/traces/services-map.png')} alt="servicemap" />
+
+The Services Map provides the following features:
+
+* Zoom in and out on the map
+* Hover over a service to see its connections to other services and the last 15 minutes of activity in terms of latency, requests, and errors
+* Click on a service to open the Entity Inspector with the ability to drill down to traces, metrics, and the service dashboard
+* Filter by application (if your tracing data has the `application=[app-name]` tag)
```

Comment:
Filter by application (if your tracing data has the `application=[app-name]` tag) and/or environment (if your tracing data has the `deployment.environment=[environment]` tag)

Reply:
thank you!